src/main/java/org/folio/orders/events/handlers/ReceiptStatusConsistency.java

```diff
@@ -60,11 +62,12 @@ public void handle(Message<JsonObject> message) {
 
     String poLineIdUpdate = messageFromEventBus.getString("poLineIdUpdate");
     String query = String.format(PIECES_ENDPOINT, poLineIdUpdate, LIMIT);
-
+    
     // 1. Get all pieces for poLineId
     getPieces(query, httpClient, okapiHeaders, logger).thenAccept(piecesCollection -> {
       List<org.folio.rest.acq.model.Piece> listOfPieces = piecesCollection.getPieces();
 
+      String lang = messageFromEventBus.getString(LANG_QUERY_PARAM);
```

Comment:
minor: use the LANG directly here instead of another variable

Reply:
Done